olmo/util.py

```diff
@@ -59,6 +59,7 @@ def __repr__(self) -> str:
 class LogFilterType(StrEnum):
     rank0_only = "rank0_only"
     local_rank0_only = "local_rank0_only"
+    firehose = "firehose"
```

Comment:
I don't have a strong opinion here but maybe "all_ranks" instead of "firehose" to be more consistent with the existing names?

Reply:
Done in [dd454bd](https://github.com/allenai/OLMo/pull/480/commits/dd454bdaced49f9809162a1f7c7d667c45a8091f).